app/src/components/Charts/StreamPerMonth/query.tsx

```diff
@@ -13,6 +12,6 @@ ORDER BY ts
 }
 
 export type QueryResult = {
-    ms_played: Float
-    ts: Date_
```

Comment:
The Date type is converted to a number (timestamp) and then handled by the plot function.